data/archived_notebook.txt

```diff
@@ -0,0 +1,2 @@
+/t JavaDocs /pin false /archive true /tag cs2113 WHITE 
+/t JavaDocs /pin false /archive true 
```

Comment:
Why is there 2 notes of the same title? This shld not be accepted when adding note.